pages/Technical Documentation/Data/Contributing to the EarthCODE Catalog.md

```diff
@@ -242,16 +252,383 @@ Alternatively, if you have access to a **reliable cloud storage service**—such
 
 If you choose to use the ESA PRR the generated STAC Items have to be sent via `POST` requests to the [PRR Registration Gateway](https://eoresults.esa.int/reg-api/docs#/) for registration using the previously requested collection.
 
-The only requirement is that __other users and the STAC browser must be able to find and read your repository!__
+The only requirement is that **other users and the STAC browser must be able to find and read your repository!**
 
 :::info  
 In the next step where you will be uploading metadata to the Open Science Catalog, EarthCODE administrators will review your Item Catalog and assist you with any necessary adjustments.  
-:::  
+:::
+
+## Step 3: Creating a Product Entry in the Open Science Catalog
+
+After completing steps 1 and 2 to prepare your data package, you're now ready to add a product entry to the Open Science Catalog (OSC).
+
+This involves creating a new collection folder under the `products` directory in the [`open-science-catalog-metadata-staging`](https://github.com/ESA-EarthCODE/open-science-catalog-metadata-staging) repository. The process is straightforward, especially if you're already familiar with Git.
```

Comment:
The URL should point the production OSC at https://github.com/ESA-EarthCODE/open-science-catalog-metadata